packages/calcite-components/src/components/input-date-picker/input-date-picker.tsx

```diff
@@ -190,6 +190,12 @@ export class InputDatePicker
   /** When `true`, interaction is prevented and the component is displayed with lower opacity. */
   @property({ reflect: true }) disabled = false;
 
+  /** Accessible placeholder for the end input component in range format. */
+  @property() endPlaceholder: string;
+
+  /** Accessible placeholder for the start input component in range format. */
+  @property() startPlaceholder: string;
+
```

Comment:
I don't think we should publicly expose placeholder properties like these since we're not going to be able to support them in a future version of this component that has input masking.  There doesn't seem to be a specific ask to add custom placeholder text based on the original issue, which is just accessible labels.

Reply:
As far as having them publicly exposed, I'll follow your advice. 

So I'm reading that Screen readers announce both the label and the placeholder when the input gains focus.
 
The way I have it set up currently, in range mode, it will say “Entering calcite-date-picker web content, Start Input (or specified label), Start Date, edit text, blank.
Vs non range “Entering calcite-date-picker web content, Date Input (or specified label), Date, edit text, blank.
 
This goes to "Choose Date" with ArrowDown to the date-picker.

Reply:
Yeah, that first one is definitely better than the second one.  I think that makes sense, plus the translation for the placeholder can be controlled by the component, keeping the burden of localizing that away from the component consumer too.

Reply:
> I don't think we should publicly expose placeholder properties like these since we're not going to be able to support them in a future version of this component that has input masking. 

If we don't support them in the future the existing `placeholder` property would need to deprecated anyway. IMO its probably ok to add these even if they will be gone in the future.

Reply:
Do we even need to have a default placeholder text? Everywhere else we don't set placeholder by default.